client/src/components/Register.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import { Input } from '@nextui-org/react';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import backgroundImage from '../img/register-splash.jpg';
+import { setUser } from '../redux/user/userSlice';
+
+export const Register = () => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+
+  const register = async (e) => {
+    e.preventDefault();
+    if (username.length >= 8 && password.length >= 8) {
+      dispatch(setUser({ username, password }));
+      localStorage.setItem('user', JSON.stringify({ username, password }));
+      navigate('/');
+    } else {
+      <p>Username and Password must be at least 8 characters long</p>;
```

Comment:
- [ ] When I press the "Register" button, this 

```} else {
      <p>Username and Password must be at least 8 characters long</p>;
```

is not being rendered on the page:

![image](https://github.com/peterdtitan/BnB-homes/assets/57421823/f43c7391-555b-441a-b211-c82578b15c1c)

Because of this, I had lots of trouble creating a new user, because I didn't know that the name and password had to be at least 8 characters.

Kindly, make sure that `<p>` is actually rendered and very clear about the minimum character length for name and password registration.

That way, your UX will be vastly improved. 👍

Reply:
Thanks for pointing this out, it has been resolved. ✅ ✅